package.json

```diff
@@ -29,7 +29,8 @@
     "clean": "git clean -fX .",
     "build": "node_modules/.bin/webpack",
     "package": "cd dist && zip -r ../lockbox-0.1pre.xpi *",
-    "prepackage": "npm run-script clean && npm run-script build"
+    "prepackage": "npm run-script clean && npm run-script build",
+    "lint": "node_modules/.bin/eslint dist/chrome/modules/aboutPage.jsm"
```

Comment:
You can just say `eslint dist/chrome/modules/aboutPage.jsm`, since npm adds `node_modules/.bin` to the `PATH`.